src/runner.py

```diff
@@ -31,6 +33,10 @@ def __init__(self, configs, args):
         runtime_config = configs['labs'][args.lab_config]
         self._runtime = kernelci.lab.get_api(runtime_config)
         self._output = args.output
+        self._ssh_host = args.ssh_host
+        self._ssh_key = args.ssh_key
+        self._ssh_port = args.ssh_port
+        self._ssh_user = args.ssh_user
```

Comment:
There's now a `kernelci.storage` package for handling uploads.  So please drop this and use it instead.  In fact `runner.py` and job templates will be updated to use it anyway, so maybe you can rebase this PR once it's done.  If you drop the `xfstests testplan integration` patch then I think this PR could be merged more easily as it would be all contained within the `fstests` directory.

Reply:
> Are we still moving on with this?

You mean running fstests?  Well I suppose so :)